bciers/apps/reporting/src/app/components/facility/FacilityEmissionAllocationForm.tsx

```diff
@@ -97,10 +133,9 @@ export default function FacilityEmissionAllocationForm({
 
   // 🔄 Check for allocation mismatch on page load to prevent submit
   useEffect(() => {
-    if (!validateEmissions(formData)) {
-      setErrors([errorMismatch]);
-    }
-    setSubmitButtonDisabled(!validateEmissions(formData));
+    const newErrors = validateFormData(formData);
+    setErrors(newErrors);
+    setSubmitButtonDisabled(newErrors.length > 0);
   }, [formData]);
```

Comment:
this one might be a question for @zoeyli-46 : Any of our other components show errors when the user tries to click 'submit', but in this case, the errors will show instantly, including on a blank form. Is this the expected behaviour?